support-frontend/assets/helpers/productPrice/weeklyProductPrice.js

```diff
@@ -22,28 +33,38 @@ const getPromotionPrice = (country: IsoCountry, period: WeeklyBillingPeriod, pro
   ].join('');
 };
 
+function getPromotion(
```

Comment:
Is there a reason to call `getPromotion` and then just calling `genericGetPromotion`? Would it be less code just to call `genericGetPromotion`?

Reply:
Yeah I went backwards and forwards with it, I think it was originally there to reduce the number of params that need to be passed but I agree that it is not really adding any value now. I'll remove it.

Reply:
Actually I'll do this in my next PR because I've already done it in that branch and don't want to redo the work.